### ISSUE TITLE
cargo install ddbug fails right now

### ISSUE THREAD

**Issue by fitzgen** [gimli-rs/ddbug#12]
Looks related to `pdb` things -- perhaps there was a breaking change that snuck in as non-breaking or something?

```
$ cargo install -f ddbug
    Updating registry `https://github.com/rust-lang/crates.io-index`
  Installing ddbug v0.1.0
 Downloading arrayvec v0.3.24
 Downloading odds v0.2.26
   Compiling odds v0.2.26
   Compiling libc v0.2.34
   Compiling scroll v0.5.0
   Compiling uuid v0.5.1
   Compiling winapi-build v0.1.1
   Compiling regex-syntax v0.3.9
   Compiling unicode-xid v0.0.4
   Compiling winapi v0.2.8
   Compiling plain v0.0.2
   Compiling nodrop v0.1.12
   Compiling fallible-iterator v0.1.3
   Compiling quote v0.3.15
   Compiling utf8-ranges v0.1.3
   Compiling byteorder v1.2.1
   Compiling log v0.3.8
   Compiling getopts v0.2.15
   Compiling synom v0.11.3
   Compiling kernel32-sys v0.2.2
   Compiling pdb v0.1.6
   Compiling memchr v0.1.11
   Compiling arrayvec v0.3.24
   Compiling syn v0.11.11
   Compiling thread-id v2.0.0
   Compiling fs2 v0.2.5
   Compiling aho-corasick v0.5.3
   Compiling thread_local v0.2.7
   Compiling gimli v0.14.0
   Compiling memmap v0.4.0
   Compiling regex v0.1.80
   Compiling scroll_derive v0.4.0
   Compiling goblin v0.0.10
   Compiling env_logger v0.3.5
   Compiling ddbug v0.1.0
error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:625:26
    |
625 |                     pdb::EnumValue::U8(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:626:26
    |
626 |                     pdb::EnumValue::U16(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:627:26
    |
627 |                     pdb::EnumValue::U32(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:628:26
    |
628 |                     pdb::EnumValue::U64(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:629:26
    |
629 |                     pdb::EnumValue::I8(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:630:26
    |
630 |                     pdb::EnumValue::I16(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:631:26
    |
631 |                     pdb::EnumValue::I32(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0433]: failed to resolve. Could not find `EnumValue` in `pdb`
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:632:26
    |
632 |                     pdb::EnumValue::I64(val) => val as i64,
    |                          ^^^^^^^^^ Could not find `EnumValue` in `pdb`

error[E0308]: mismatched types
  --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:98:52
   |
98 |                 let return_type = parse_type_index(return_type);
   |                                                    ^^^^^^^^^^^ expected u32, found enum `std::option::Option`
   |
   = note: expected type `u32`
              found type `std::option::Option<u32>`
   = help: here are some functions which might fulfill your needs:
           - .unwrap()
           - .unwrap_or_default()

error[E0308]: mismatched types
   --> /home/fitzgen/.cargo/registry/src/github.com-1ecc6299db9ec823/ddbug-0.1.0/src/pdb.rs:121:58
    |
121 |                 let this_pointer_type = parse_type_index(this_pointer_type);
    |                                                          ^^^^^^^^^^^^^^^^^ expected u32, found enum `std::option::Option`
    |
    = note: expected type `u32`
               found type `std::option::Option<u32>`
    = help: here are some functions which might fulfill your needs:
            - .unwrap()
            - .unwrap_or_default()

error: aborting due to 10 previous errors

error: failed to compile `ddbug v0.1.0`, intermediate artifacts can be found at `/tmp/cargo-install.W6V5RnUkIBZg`

Caused by:
  Could not compile `ddbug`.

To learn more, run the command again with --verbose.
```

**Comment by fitzgen** [comment]
I *think* `pdb::EnumValue` became `pdb::Variant`?

**Comment by philipc** [comment]
Yeah it did. I've fixed that in master, but still have the panopticon dependency so can't publish it yet. I can probably fix it on a branch and publish that. You could also try:
`cargo install --git https://github.com/gimli-rs/ddbug`

**Comment by philipc** [comment]
Published 0.2.0, using master with the panopticon dependency removed.
